po/es.po

```diff
@@ -200,6 +197,10 @@ msgid ""
 "domains like i❤tacos.ws will show up properly in the URL entry, while "
 "phishing domains should still show up like xn--80ak6aa92e.com"
 msgstr ""
+"🎉️ ¡Soporte de emoticonos (y caracteres internacionales) para dominios! "
+"Punycode significa que dominios como i❤tacos.ws se mostrarán correctamente en "
+"la entrada de la URL, mientras que los dominios de phishing aún deberían "
```

Comment:
“URL” is a masculine noun in Spanish; if you expand the abbreviation you’ll notice (_localiza**dor** uniforme de recursos_). “Phishing” should be written with quotation marks («phishing») because it’s a foreign word.